ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/TestJwtVerifierLoader.scala

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.sandbox
+
+import com.daml.jwt.{HMAC256Verifier, JwtVerifier}
+import com.daml.ledger.api.auth.JwtVerifierLoader
+import com.daml.ledger.api.domain.JwksUrl
+import com.daml.platform.sandbox.TestJwtVerifierLoader._
+
+import scala.concurrent.Future
+
+class TestJwtVerifierLoader(defaultHMACSecret: String) extends JwtVerifierLoader {
+  private def verifier(secret: String): JwtVerifier =
+    HMAC256Verifier(secret).getOrElse(sys.error("Failed to create HMAC256 verifier"))
+
+  private val defaultJwtVerifier = verifier(defaultHMACSecret)
+
+  private val verifierMap = Map(
+    jwksUrl1 -> Future.successful(verifier(defaultHMACSecret)),
+    jwksUrl2 -> Future.successful(verifier(secret2)),
+    jwksUrl3 -> Future.failed(new VerifierException),
+  )
+
+  override def loadJwtVerifier(
+      jwksUrl: JwksUrl,
+      keyId: Option[String],
+  ): Future[JwtVerifier] = verifierMap.getOrElse(jwksUrl, Future.successful(defaultJwtVerifier))
+}
+
+object TestJwtVerifierLoader {
+  val jwksUrl1: JwksUrl = JwksUrl.assertFromString("http://daml.com/jwks1.json")
```

Comment:
Off topic, but my expectation of `JwksUrl.fromString` was that the JSON was passed in the string (as opposed to a url).  Not sure if this would be better named as `assertFromUrl` or `assertFromUrlString`

Reply:
Will consider for the cleanup